src/style.css

```diff
@@ -1,3 +1,81 @@
+
+html {
+  font-family: Roboto, sans-serif;
+
+  --blue-link:#00ACDC;
+}
+
 body {
   margin: 0;
 }
+
+nav {
```

Comment:
In general try using classes and also organize your code following the hierarchy